apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/EventActivity.java

```diff
@@ -188,7 +188,7 @@ private boolean onlyStringProperties() {
     }
 
     @Override
-    protected void send(View view) {
+    public void send(View view) {
```

Comment:
old VM could not run tests anymore for some reason (even on develop: https://msmobilecenter.visualstudio.com/SDK/_build/results?buildId=833380).
Switch to another pool and got the new lint error, which means lint behavior depends on what is installed on the machine for some reason.